Add configuration details for TLS DB connection (#333)
Enalean/tuleap-documentation-en: languages/en/installation-guide/docker-image.rst

* The database admin user (root or equivalent) as ``DB_ADMIN_USER`` environment variable
* The database admin user password as ``DB_ADMIN_PASSWORD`` environment variable

You can also modify the behaviour of Tuleap with the following environment variables:

* ``TULEAP_DB_SSL_MODE``: by default, the connection to the database is not encrypted. You can change that to either:

>   * ``no-verify``: connection is encrypted but host certificate is not checked. Your traffic cannot be captured but you are vulnerable to Man In The Middle attacks.

Reviewer: ```suggestion
  * ``no-verify``: connection is encrypted but host certificate is not checked. Your traffic cannot be passively inspected but you are vulnerable to Man In The Middle attacks.
```

  * ``verify-ca``: connection is encrypted and host certificate is verified. Your db server must use a public trusted certificate. **WARNING**: perl & python code (used for subversion) cannot enforce this, those parts will do encryption without certificate verification.

* ``TULEAP_DB_AZURE_SUFFIX``: only needed if you deploy on Microsoft Azure MySQL. It should correspond to first part of ``DB_HOST`` (see `official documentation <https://docs.microsoft.com/en-us/azure/mysql/quickstart-create-mysql-server-database-using-azure-portal#connect-to-mysql-by-using-the-mysql-command-line-tool>`_)
* ``TULEAP_FPM_SESSION_MODE``: you can set it to ``redis`` so php sessions will be stored in a `Redis <https://redis.io>`_ K/V store. This also activate usage of redis for Tuleap (background events, etc).
* ``TULEAP_REDIS_SERVER``: needed if you set ``redis`` for ``TULEAP_FPM_SESSION_MODE``
* ``TULEAP_REDIS_PORT``: needed if redis is listening on port that is not ``6379`` (the default)
* ``TULEAP_REDIS_PASSWORD``: needed if redis requires a password

> * ``TULEAP_REDIS_USE_TLS``: set to 1 if you want encrypt connection to redis

Reviewer: ```suggestion
* ``TULEAP_REDIS_USE_TLS``: set to 1 if you want to encrypt the connection to Redis
```

* ``443`` TCP https traffic
* ``22`` TCP ssh traffic (for git)

Certificates
````````````

Tuleap container generate a self signed certificate can only communicate in https. In order to have a valid certificate
for your end users you either need to:

* Delegate the certificate management to your ingress controller (Kubernetes)
* Have a reverse proxy in front of the Tuleap container to deal with the certificate
* I you don't want to bother with a reverse proxy, you can modify the certificate generated by tuleap and set yours:

  * They key must be ``/data/etc/pki/tls/private/localhost.key.pem``

>   * The cert must be ``/data//etc/pki/tls/certs/localhost.cert.pem``

Reviewer: ```suggestion
  * The cert must be ``/data/etc/pki/tls/certs/localhost.cert.pem``
```
